[2020-10-13 15:00:21,693][http-nio-8081-exec-4][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:102] org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'exercises_type' cannot be null
### The error may exist in file [/Users/zhangqiang/Documents/04-Project/BSS/SourceCode/shangshufang.apiservice/target/classes/mapping/ExerciseWarehouseKnowledgeChoiceQuestionMapping.xml]
### The error may involve com.shangshufang.apiservice.mapper.ExerciseWarehouseKnowledgeChoiceQuestionMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO shangshufang_base.exercise_warehouse_knowledge_choice_question     (       technology_id,       knowledge_id,       exercises_title,       exercises_source,       exercises_type,       create_user,       update_user      )     VALUES      (       ?,       ?,       ?,       ?,       ?,       ?,       ?       )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'exercises_type' cannot be null
; SQL []; Column 'exercises_type' cannot be null; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'exercises_type' cannot be null
[2020-10-13 15:15:53,473][http-nio-8081-exec-3][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:102] org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'exercises_type' cannot be null
### The error may exist in file [/Users/zhangqiang/Documents/04-Project/BSS/SourceCode/shangshufang.apiservice/target/classes/mapping/ExerciseWarehouseKnowledgeChoiceQuestionMapping.xml]
### The error may involve com.shangshufang.apiservice.mapper.ExerciseWarehouseKnowledgeChoiceQuestionMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO shangshufang_base.exercise_warehouse_knowledge_choice_question     (       technology_id,       knowledge_id,       exercises_title,       exercises_source,       exercises_type,       create_user,       update_user      )     VALUES      (       ?,       ?,       ?,       ?,       ?,       ?,       ?       )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'exercises_type' cannot be null
; SQL []; Column 'exercises_type' cannot be null; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'exercises_type' cannot be null
[2020-10-13 15:19:51,435][http-nio-8081-exec-6][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:102] org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'data_status' doesn't have a default value
### The error may exist in file [/Users/zhangqiang/Documents/04-Project/BSS/SourceCode/shangshufang.apiservice/target/classes/mapping/ExerciseWarehouseKnowledgeChoiceQuestionMapping.xml]
### The error may involve com.shangshufang.apiservice.mapper.ExerciseWarehouseKnowledgeChoiceQuestionMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO shangshufang_base.exercise_warehouse_knowledge_choice_question     (       technology_id,       knowledge_id,       exercises_title,       exercises_source,       exercises_type,       create_user,       update_user      )     VALUES      (       ?,       ?,       ?,       ?,       ?,       ?,       ?       )
### Cause: java.sql.SQLException: Field 'data_status' doesn't have a default value
; SQL []; Field 'data_status' doesn't have a default value; nested exception is java.sql.SQLException: Field 'data_status' doesn't have a default value
[2020-10-13 15:21:16,744][http-nio-8081-exec-8][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:102] org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'data_status' doesn't have a default value
### The error may exist in file [/Users/zhangqiang/Documents/04-Project/BSS/SourceCode/shangshufang.apiservice/target/classes/mapping/ExerciseWarehouseKnowledgeChoiceQuestionMapping.xml]
### The error may involve com.shangshufang.apiservice.mapper.ExerciseWarehouseKnowledgeChoiceQuestionMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO shangshufang_base.exercise_warehouse_knowledge_choice_question     (       technology_id,       knowledge_id,       exercises_title,       exercises_source,       exercises_type,       create_user,       update_user      )     VALUES      (       ?,       ?,       ?,       ?,       ?,       ?,       ?       )
### Cause: java.sql.SQLException: Field 'data_status' doesn't have a default value
; SQL []; Field 'data_status' doesn't have a default value; nested exception is java.sql.SQLException: Field 'data_status' doesn't have a default value
[2020-10-13 15:24:41,677][http-nio-8081-exec-10][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:102] java.lang.NullPointerException
[2020-10-13 15:28:15,423][http-nio-8081-exec-1][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:103] org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'technology_id' doesn't have a default value
### The error may exist in file [/Users/zhangqiang/Documents/04-Project/BSS/SourceCode/shangshufang.apiservice/target/classes/mapping/ExerciseWarehouseKnowledgeChoiceQuestionOptionMapping.xml]
### The error may involve com.shangshufang.apiservice.mapper.ExerciseWarehouseKnowledgeChoiceQuestionOptionMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO shangshufang_base.exercise_warehouse_knowledge_choice_option_question     (       exercises_id,       option_text,       is_answer,       create_user,       update_user      )     VALUES      (       ?,       ?,       ?,       ?,       ?       )
### Cause: java.sql.SQLException: Field 'technology_id' doesn't have a default value
; SQL []; Field 'technology_id' doesn't have a default value; nested exception is java.sql.SQLException: Field 'technology_id' doesn't have a default value
[2020-10-13 15:34:02,985][http-nio-8081-exec-2][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:62] org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.binding.BindingException: Parameter 'dataStatus' not found. Available parameters are [knowledgeID, exercisesID, param3, technologyID, param1, param2]
[2020-10-13 15:35:39,738][http-nio-8081-exec-3][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:62] org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.binding.BindingException: Parameter 'dataStatus' not found. Available parameters are [knowledgeID, exercisesID, param3, technologyID, param1, param2]
[2020-10-13 16:38:32,360][http-nio-8081-exec-8][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:133] org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.reflection.ReflectionException: There is no getter for property named 'majorName' in 'class com.shangshufang.apiservice.entity.ExerciseWarehouseKnowledgeChoiceQuestionEntity'
[2020-10-13 16:40:19,640][http-nio-8081-exec-9][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:133] org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.reflection.ReflectionException: There is no getter for property named 'majorName' in 'class com.shangshufang.apiservice.entity.ExerciseWarehouseKnowledgeChoiceQuestionEntity'
[2020-10-13 16:40:32,956][http-nio-8081-exec-1][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:133] org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.reflection.ReflectionException: There is no getter for property named 'majorName' in 'class com.shangshufang.apiservice.entity.ExerciseWarehouseKnowledgeChoiceQuestionEntity'
[2020-10-13 16:42:04,059][http-nio-8081-exec-2][ERROR][com.shangshufang.apiservice.service.impl.ExerciseWarehouseKnowledgeChoiceQuestionServiceImpl:133] org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.reflection.ReflectionException: There is no getter for property named 'majorName' in 'class com.shangshufang.apiservice.entity.ExerciseWarehouseKnowledgeChoiceQuestionEntity'
